website/public/locales/fr/translation.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
this is meant to be in a different branch and pull request, since it's unrelated to the issue 100

Reply:
I should move it to master branch and pull request to the forked repo (Idrinth/api-bench) I guess right

Reply:
no, move the branch to a new branch, not master.